store/tikv/oracle/oracles/pd.go

```diff
@@ -35,8 +35,9 @@ const slowDist = 30 * time.Millisecond
 type pdOracle struct {
 	c pd.Client
 	// txn_scope (string) -> lastTSPointer (*uint64)
-	lastTSMap sync.Map
-	quit      chan struct{}
+	lastTSMap     sync.Map
+	lastArrivalTS uint64
```

Comment:
maybe directly use `lastArrivalTime time.Time`？we only need the relative time in the local machine.

Reply:
Timestamp can simply convert to time.Time use oracle.GetTimeFromTS.